src/App.jsx

```diff
@@ -14,57 +16,60 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {selectedGood === null ? (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {`${selectedGood} is selected`}
+          <button
+            onClick={() => setSelectedGood(null)}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        </h1>
+      )}
+      <table className="table">
+        <tbody>
+          {goods.map(good => {
+            return (
+              <tr
+                data-cy="Good"
+                key={good}
+                className={classNames({
+                  'has-background-success-light': good === selectedGood,
+                })}
+              >
+                <td>
+                  <button
+                    onClick={() => {
```

Comment:
it may pass test if you include two separate button tags for adding and removing, ensuring that each is displayed  according to state

Reply:
In this case, I believe that this is an unnecessary code deduction. But in the case of a real project, this is the case, since a button can have many more nested elements and properties